src/components/BulkEnrollmentPage/CourseSearchResultsCells.test.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { CourseNameCell, FormattedDateCell } from './CourseSearchResultsCells';
+
+const testCourseName = 'TestCourseName';
+const testCourseRunKey = 'TestCourseRun';
+const testStartDate = '2020-09-10T04:00:00Z';
+
+describe('CourseNameCell', () => {
+  const row = {
+    original: {
+      key: testCourseRunKey,
+    },
+  };
+  const slug = 'sluggy';
+  const wrapper = mount(<CourseNameCell value={testCourseName} row={row} enterpriseSlug={slug} />);
+  it('correctly formats a link', () => {
+    expect(wrapper.find('a').props().href).toEqual(`http://localhost:8734/${slug}/course/${row.original.key}`);
```

Comment:
forgot where this base url is setup, can this be a constant, or exported from somewhere?

Reply:
yeah, it's probably a constant, I can change that.